packages/gafl-webapp-service/src/pages/summary/contact-summary/__tests__/contact-summary.spec.js

```diff
@@ -99,6 +80,8 @@ describe('The contact summary page', () => {
     const data = await injectWithCookies('GET', CONTACT_SUMMARY.uri)
     expect(data.statusCode).toBe(302)
     expect(data.headers.location).toBe(NAME.uri)
+    const data2 = await injectWithCookies('GET', NAME.uri)
+    expect(data2.payload.search(backLinkRegEx(LICENCE_SUMMARY.uri)) > 0).toBeTruthy()
```

Comment:
This looks like it should be a separate, new test

Reply:
done